Tasks/ServiceFabricComposeDeploy/ServiceFabricComposeDeploy.ps1

```diff
@@ -117,7 +143,14 @@ try {
         {
             Write-Host (Get-VstsLocString -Key CurrentStatus -ArgumentList $existingApplication.ComposeApplicationStatus)
             Start-Sleep -Seconds 3
-            $existingApplication = Get-ServiceFabricComposeApplicationStatusPaged @getStatusParameters
+            if ($usePreviewApi)
```

Comment:
This if-else with version is in multiple places. Consider making it a function which takes isPreviewVersion param.